package.json

```diff
@@ -46,6 +46,7 @@
     "eslint-plugin-prettier": "^3.1.3",
     "eslint-plugin-simple-import-sort": "^5.0.3",
     "express": "^4.17.1",
+    "fastify": "^2.14.1",
```

Comment:
could you verify with `fastify@3` as well?

Reply:
Done